src/test/ui/span/issue-35987.stderr

```diff
@@ -1,10 +1,10 @@
-error[E0404]: `Add` is not a trait
+error[E0404]: expected trait, found type parameter `Add`
   --> $DIR/issue-35987.rs:15:21
    |
 15 | impl<T: Clone, Add> Add for Foo<T> {
-   |                ---  ^^^ expected trait, found type parameter
-   |                |
-   |                type parameter defined here
+   |                     ^^^ not a trait
+   |
+   = help: possible better candidate is found in another module, you can import it into scope: `use std::ops::Add;`
```

Comment:
Seems the label got worse here, but I like the help that was added.  A combination might be better.